pkg/controller/controller_test.go

```diff
@@ -1503,6 +1504,8 @@ func TestProvisionFromSnapshot(t *testing.T) {
 				},
 			},
 			snapshotStatusReady: true,
+			notPopulated:        true,
+			expectErr:           true,
```

Comment:
Do we have a success test case?

Reply:
because 

> Do we have a success test case?

In default, the volume content source not be populated, why?

Reply:
By default the volume content source will not be populated because it will be an empty volume.

Reply:
This should be a success test case.  Volume content source should be populated.  This seems to be wrong.

Reply:
In the success test case, Actually  the volume content source not be populated, why?

Reply:
The mock server used for unit testing needs to be updated. See https://github.com/kubernetes-csi/external-provisioner/blob/a297985bed50f97901b869558ceb099b286ea544/pkg/controller/controller_test.go#L416